architecture/index.adoc

```diff
@@ -0,0 +1,70 @@
+:context: architecture-overview
+
+toc::[]
+
+{product-title} is a cloud-based Kubernetes container platform.
+The foundation of {product-title} is based on Kubernetes and therefore shares the same technology.
+To learn more about {product-title} and Kubernetes, refer to the xref:../architecture/architecture.adoc#architecture[product architecture].
+
+[id="about-installation-and-updates"]
+== About installation and updates
+
+As a cluster administrator, you can use the {product-title} xref:../architecture/architecture-installation.adoc#architecture-installation[installation program] to install and deploy a cluster by using one of the following methods:
+
+** Installer-provisioned infrastructure (IPI)
+** User-provisioned infrastructure (UPI)
+
+[id="about-control-planes"]
+== About control planes
+
+The xref:../architecture/control-plane.adoc#control-plane[control plane] manages the worker nodes and the pods in your cluster. You can configure nodes with the use of machine config pools (MCPs).
+MCPs are groups of machines, such as control plane components or user workloads, that are based on the resources that they handle.
+{product-title} assigns different roles to hosts. These roles define the function of a machine in a cluster.
+The cluster contains definitions for the standard control plane and worker role types.
+
+You can use Operators to package, deploy, and manage services on the control plane. With Operators, you can use `kubectl` and `oc` commands.
+Operators are important components in {product-title} because they provide the following services:
+
+* Perform health checks
+* Enable the way to watch an application
+* Manage over-the-air updates
+* Ensure applications stay in the specified state
+
+[id="about-containerized-applications-for-developers"]
+== About containerized applications for developers
+
+As a developer, you can use different tools, methods, and formats to xref:../architecture/understanding-development.adoc#understanding-development[develop your containerized application] based on your unique requirements, for example:
+
+- Use various build-tool, base-image, and registry options to build a simple container application.
+- Use supporting components such as OperatorHub and templates to develop your application.
+- Package and deploy your application as an Operator.
+
+You can also create a Kubernetes manifest and store it in a Git repository.
+Kubernetes works on basic units called pods. A pod is a single instance of a running process in your cluster. Pods can contain one or more containers.
+You can create a service by grouping a set of pods and their access policies.
+Services provide permanent internal IP addresses and host names for other applications to use as pods are created and destroyed. Kubernetes defines workloads based on the type of your application.
+
+[id="coreos-and-ignition"]
+== About {op-system-first} and Ignition
+
+As a cluster administrator, you can perform the following tasks:
```

Comment:
Should this specify that the tasks are related to RHCOS?

Reply:
+1